### ISSUE TITLE
visualization

### ISSUE THREAD

**Issue by tmz153** [ZYK100/LLCM#29]
Excuse me, how do you get the initial intra-class and inter-class distances, thanks!  Need model and parameters?

**Comment by ZYK100** [comment]
Hello, the initial inter class and intra class distances are extracted by a pre-trained ResNet-50 without any fine-tuning.

**Comment by tmz153** [comment]
Thanks, I also used only the pre-training model parameters from resnet-50, but I kept the structural form of agw (no nolocal). Is this approach not correct?My resultant plots are around 0.3, I don't know why? Trouble again.
![B`J4BFS((VN%@FNCMXKVPUJ](https://github.com/user-attachments/assets/a1be3e34-b9ca-450b-aefa-d614a5f48820)

**Comment by ytzxc** [comment]
hello, did you resolve this problem,i cannot get the initial distribution result?

**Comment by ccccxxxhhh299** [comment]
> 谢谢，我也只使用了 resnet-50 中的预训练模型参数，但我保留了 agw 的结构形式（没有 nolocal）。这种方法不正确吗？我得到的图在 0.3 左右，我不知道为什么？又麻烦了。 ![B`J4BFS((VN%@FNCMXKVPUJ](https://private-user-images.githubusercontent.com/105350036/365514035-a1be3e34-b9ca-450b-aefa-d614a5f48820.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzEwMzU5MzMsIm5iZiI6MTczMTAzNTYzMywicGF0aCI6Ii8xMDUzNTAwMzYvMzY1NTE0MDM1LWExYmUzZTM0LWI5Y2EtNDUwYi1hZWZhLWQ2MTRhNWY0ODgyMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwOFQwMzEzNTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xYjdmMjMwNDRlNTIwZmMzNTNjNGM4OTIyNDBhOWEzYzM2MDk1ZmRjZjhlNDgxZDUxMDNiNjVhZWQ1NDhiOTg1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.hPz9dJcWuut0FblT-A-v1aYLcjYQpo8MeQtBRW6pKVs)

Hello, I have the same question. How did you solve it?

**Comment by ytzxc** [comment]
no, I give up this visualization. I do not know how to achieve this. If you answer this question, please tell me. Thank you!
&nbsp;
&nbsp;
------------------&nbsp;Original&nbsp;------------------
From: ***@***.***&gt;;
Date: &nbsp;Fri, Nov 8, 2024 11:40 AM
To: ***@***.***&gt;; 
Cc: ***@***.***&gt;; ***@***.***&gt;; 
Subject: &nbsp;Re: [ZYK100/LLCM] visualization (Issue #29)

&nbsp;



  
谢谢，我也只使用了 resnet-50 中的预训练模型参数，但我保留了 agw 的结构形式（没有 nolocal）。这种方法不正确吗？我得到的图在 0.3 左右，我不知道为什么？又麻烦了。 
  
Hello, I have the same question. How did you solve it?
